src/useInterval.ts

```diff
@@ -1,17 +1,18 @@
 import { useEffect, useRef } from 'react';
 
 const useInterval = (callback: Function, delay?: number | null) => {
-  const latestCallback = useRef<Function>(() => {});
+  const savedCallback = useRef<Function>(() => {});
 
   useEffect(() => {
-    latestCallback.current = callback;
+    savedCallback.current = callback;
   });
```

Comment:
I think this effect needs a second argument. Here is a detailed comment from [another pull request](https://github.com/streamich/react-use/pull/620#issuecomment-536245044).

```suggestion
  }, [callback]);
```

Reply:
See https://github.com/streamich/react-use/pull/626/files#issuecomment-541392191